src/python/BFinitPassthrough.py

```diff
@@ -126,6 +126,9 @@ def reset_to_bootloader(args):
     else:
         BootloaderInitSeq = bootloader.get_init_seq('CRSF', args.type)
         dbg_print("  * Using full duplex (CRSF)")
+        #this is the training sequ for the ROM bootloader, we send it here so it doesn't auto-neg to the wrong baudrate by the BootloaderInitSeq that we send to reset ELRS
```

Comment:
Is the indentation wrong?

Reply:
I don't understand the issue is

Reply:
Actually, that makes sense to have it only in case of full duplex uart. 
Ignore that comment...